README.md

```diff
@@ -170,7 +172,7 @@ If you would like to generate (still under development) installer scripts, you c
 to either `init` or `generate-ci`:
 
 ```sh
-cargo dist init --ci=github --installer=github-shell --installer=github-powershell
+cargo dist init --ci=github --installer=shell --installer=powershell
```

Comment:
does clap let you do --installer=shell,powershell to mean the same as this

Reply:
(I also explicitly don't add short forms, again because stuff like `curl -LsSf` fills me with SADNESS even though it's nice that it's compact for installer hints)

Reply:
(fight me)